imap_processing/cdf/config/imap_swe_global_cdf_attrs.yaml

```diff
@@ -0,0 +1,25 @@
+instrument_base: &instrument_base
+  Descriptor: SWE>Solar Wind Electron
+  TEXT: >
+    The IMAP Solar Wind Electron (SWE) instrument measures the solar wind electrons
+    in the heliosphere. SWE will contribute to our understanding of the acceleration
+    and transport of charged particles in the heliosphere.
+    SWE design and assembly is led by the Princeton Plasma Physics Laboratory. See
+    https://imap.princeton.edu/instruments/swe for more details.
+  Data_version: "001"
+  Instrument_type: "Particles (space)"
+
+imap_swe_l1a_sci:
+  <<: *instrument_base
+  # NOTE: Right now, this Data_level is required to produce valid CDF
```

Comment:
Was that introduced in a new version of `cdflib`? I think I missed that.

Reply:
This is introduced by both our schema usage and `cdflib` version we are using. New `cdflib` catches this error. What's happening behind the scene was that new schema sorts global attrs in alphebetic order and it set those key that we don't set value for to `None` value. Then `cdflib` version we have doesn't catch this `None` value. If a global attrs is set to `None`, `cdflib`skips adding remaining global attrs.